livre/eq/Config.cpp

```diff
@@ -158,7 +155,10 @@ void Config::unmapFrameData()
 
 void Config::resetCamera()
```

Comment:
reset for me has by definition side effects, and the publish in other places where the camera is changed is implicit.

For the sake of this PR I would keep at is, we can discuss this later and address it consistently.